zerver/webhooks/facebook/doc.md

```diff
@@ -0,0 +1,3 @@
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
```

Comment:
Now, I would recommend studying the documentation for other integrations and then writing the contents of the `doc.md` file. And claim the documentation task for the Facebook integration. We can't merge the PR till it is documented! :)

Reply:
And don't worry about circleci, it is not because of your PR! :)

Reply:
@nyan-salmon This is the task for adding documentation: [5647007018385408](https://codein.withgoogle.com/tasks/5647007018385408), I think it would be perfect for you. But be sure to claim it soon, the claim new task deadline is just around the corner :)

Reply:
@eeshangarg @alenavolk Thank you for advices!
I will claim that task!